debian/control

```diff
@@ -9,12 +9,10 @@ Standards-Version: 3.9.8
 Build-Depends: debhelper (>= 9),
   cmake,
   doxygen,
-#  This package isn't available in older Ubuntu releases.
-#  As long as we support 14.04, we build using the embedded one.
-#  libminizip-dev,
   libasio-dev,
   libglew-dev,
   libicu-dev,
+  libminizip-dev,
```

Comment:
Does this automatically add `libminizip` to the run/install dependencies or is this configured somewhere else?

Reply:
@matthiakl I think that happens in the 'Depends' field where `${shlibs:Depends}` is used.
https://www.debian.org/doc/manuals/maint-guide/dreq.en.html